.goreleaser.yml

```diff
@@ -1,23 +1,4 @@
-project_name: tfinstall
-builds:
-  - env:
-      - CGO_ENABLED=0
-    main: ./cmd/tfinstall/main.go
-    mod_timestamp: '{{ .CommitTimestamp }}'
-    id: "tfinstall"
-    binary: tfinstall
-    flags:
-      - -trimpath
-    ldflags:
-    - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
-    goos:
-      - linux
-      - darwin
-      - windows
-    goarch:
-      - amd64
-      - arm
-      - arm64
+project_name: terraform-exec
```

Comment:
I'm not sure the config is useful without the `builds` section - maybe it's worth just removing it entirely?

Reply:
The only thing it does now is create the GitHub release, which I agree is probably not that useful.

Reply:
Release with what artifacts/assets? Just archive of source code?

Reply:
Yes

Reply:
I think we should remove it then and avoid publishing releases if there's no "real" artifact to publish - to signal that this is a library to be consumed via Go.